### PR TITLE
Consolidate etcd upgrade

### PR DESCRIPTION
**Pull request by ingvagabund**
Consolidates `etcd_upgrade` into the `etcd` role.

### CODE REVIEW

File: roles/etcd_common/defaults/main.yml [hunk 1]
```diff
@@ -73,3 +73,6 @@ etcd_advertise_client_urls: "{{ etcd_url_scheme }}://{{ etcd_ip }}:{{ etcd_clien
 etcd_listen_client_urls: "{{ etcd_url_scheme }}://{{ etcd_ip }}:{{ etcd_client_port }}"
 
 etcd_systemd_dir: "/etc/systemd/system/{{ etcd_service }}.service.d"
+
+etcd_peer: 127.0.0.1
```
[the remark this answers is not in the document]
Comment (ingvagabund) [reply roles/etcd_common/defaults/main.yml]: PR updated, put a comment telling the `etcd_peer` must be set by the role caller.

**Comment by sdodson** [comment]
shouldn't have any merge conflicts from this one
/lgtm

**Pull request merged**